geniza/settings/components/base.py

```diff
@@ -251,6 +252,11 @@
 # exclude admin and cms urls from csp directives since they're authenticated
 CSP_EXCLUDE_URL_PREFIXES = ("/admin", "/cms")
 
+# use jpg instead of png since some providers only support jpg
+DJIFFY_THUMBNAIL_FORMAT = "jpg"
+# disable djiffy import check, since we are not using djiffy views
+# DJIFFY_IMPORT_CHECK_SUPPORTED = False
```

Comment:
Should this be uncommented or is it a dev only thing?

Reply:
I fixed (well, changed) something in djiffy so it isn't needed right now — but I'm not sure the change I made in djiffy is actually the right one! which is why I left it. djiffy includes some views that assume book format paginated content, but we're not using them — so it might make sense to keep those supported checks as they were before and opt out of them in projects like this one where we don't care about those views.